docs/start/getting-started/fragments/angular/setup.md

```diff
@@ -3,7 +3,7 @@
 Use the [Angular CLI](https://github.com/angular/angular-cli) to bootstrap a new Angular app:
 
 ```bash
-npx ng new amplify-app
+npx -p @angular/cli ng new amplify-app
```

Comment:
@aspittel is this after installing `npm install -g @angular/cli` ? that this happens?

Reply:
No -- the instructions I believe are skipping over installing the amplify CLI and using `npx` instead.

Reply:
As per our offline convo, I will approve this